systemtest/src/test/java/io/strimzi/systemtest/security/custom/CustomCaST.java

```diff
@@ -68,6 +69,7 @@ public class CustomCaST extends AbstractST {
     private static final String STRIMZI_INTERMEDIATE_CA = "C=CZ, L=Prague, O=Strimzi, CN=StrimziIntermediateCA";
 
     @ParallelNamespaceTest
+    @KRaftNotSupported
```

Comment:
Description why it is not supported would be nice

Reply:
Done.